<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>download_info</class>
 <widget class="QWidget" name="download_info">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>766</width>
    <height>153</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>766</width>
    <height>153</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>766</width>
    <height>153</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Download Information</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/icon/icon/logo.png</normaloff>:/icon/icon/logo.png</iconset>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>46</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Category :</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cb_category">
   <property name="geometry">
    <rect>
     <x>106</x>
     <y>44</y>
     <width>171</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_add_category">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>44</y>
     <width>31</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Save As :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_saveas">
   <property name="geometry">
    <rect>
     <x>107</x>
     <y>78</y>
     <width>521</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_browse">
   <property name="geometry">
    <rect>
     <x>632</x>
     <y>78</y>
     <width>31</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_download_later">
   <property name="geometry">
    <rect>
     <x>106</x>
     <y>114</y>
     <width>161</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Download Later</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_start_download">
   <property name="geometry">
    <rect>
     <x>304</x>
     <y>114</y>
     <width>161</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Start Download</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_cancel">
   <property name="geometry">
    <rect>
     <x>502</x>
     <y>114</y>
     <width>161</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_size">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>90</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>64</x>
     <y>5</y>
     <width>601</width>
     <height>29</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>URL :</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="le_url">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btn_reload_categor_list">
   <property name="geometry">
    <rect>
     <x>313</x>
     <y>45</y>
     <width>91</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Reload list</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
